pkg/transform/kubernetes/kubernetes.go

```diff
@@ -106,6 +106,12 @@ func (t *Transformer) CreateDeployments(o *object.Service) ([]runtime.Object, er
 		},
 	}
 
+	// Set replica value only if it is given
+	if o.Replicas > 0 {
+		replicas := o.Replicas
+		d.Spec.Replicas = &replicas
```

Comment:
This is a better thing to do or should I do: `d.Spec.Replicas = &o.Replicas` ?
Thoughts: @tnozicka @containscafeine ?